
# Turn off use win32.coff
# 
# To apply this patch:
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'applypatch' program with this patch file as input.
#
# If you do not have 'applypatch', it is part of the 'makepatch' package
# that you can fetch from the Comprehensive Perl Archive Network:
# http://www.perl.com/CPAN/authors/Johan_Vromans/makepatch-x.y.tar.gz
# In the above URL, 'x' should be 2 or higher.
#
# To apply this patch without the use of 'applypatch':
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'patch' program with this file as input.
#
#### End of Preamble ####

#### Patch data follows ####
diff -c "C:\strawberry\cpanplus -VVM\00_--_PAR-Packer-1.009\myldr\Makefile.PL" "C:\strawberry\cpanplus\5.12.3\build\PAR-Packer-1.009\myldr\Makefile.PL"
Index: ./myldr/Makefile.PL
*** ./myldr/Makefile.PL	Tue Mar 22 19:53:31 2011
--- ./myldr/Makefile.PL	Tue Jun 14 16:49:32 2011
***************
*** 104,110 ****
      $ccdebug = $debug ? '-g ' : '';
      $lddebug = ($debug or $^O eq 'darwin') ? '' : '-s ';
      $warn = $debug ? '-Wall -Wno-comments ' : '';
!     $res = ($^O =~ /^(?:MSWin|cygwin)/) ? 'win32.coff' : '';
      $long_literal = '-l';
      $mt_cmd = '-$(NOOP)';
  } else {
--- 104,111 ----
      $ccdebug = $debug ? '-g ' : '';
      $lddebug = ($debug or $^O eq 'darwin') ? '' : '-s ';
      $warn = $debug ? '-Wall -Wno-comments ' : '';
!     $res =  ''; # VVM
! #    $res = ($^O =~ /^(?:MSWin|cygwin)/) ? 'win32.coff' : '';
      $long_literal = '-l';
      $mt_cmd = '-$(NOOP)';
  } else {
#### End of Patch data ####

#### ApplyPatch data follows ####
# Data version        : 1.0
# Date generated      : Tue Jun 14 16:59:06 2011
# Generated by        : makepatch 2.04
# Recurse directories : Yes
# Excluded files      : (\A|/).*\~\Z
#                       (\A|/).*\.a\Z
#                       (\A|/).*\.bak\Z
#                       (\A|/).*\.BAK\Z
#                       (\A|/).*\.elc\Z
#                       (\A|/).*\.exe\Z
#                       (\A|/).*\.gz\Z
#                       (\A|/).*\.ln\Z
#                       (\A|/).*\.o\Z
#                       (\A|/).*\.obj\Z
#                       (\A|/).*\.olb\Z
#                       (\A|/).*\.old\Z
#                       (\A|/).*\.orig\Z
#                       (\A|/).*\.rej\Z
#                       (\A|/).*\.so\Z
#                       (\A|/).*\.Z\Z
#                       (\A|/)\.del\-.*\Z
#                       (\A|/)\.make\.state\Z
#                       (\A|/)\.nse_depinfo\Z
#                       (\A|/)core\Z
#                       (\A|/)tags\Z
#                       (\A|/)TAGS\Z
# p "myldr/Makefile.PL" 10179 1308059372 0100666
#### End of ApplyPatch data ####

#### End of Patch kit [created: Tue Jun 14 16:59:06 2011] ####
#### Patch checksum: 56 2184 611 ####
#### Checksum: 74 2829 54096 ####
